karma.options.js

```diff
@@ -170,6 +170,7 @@ const options = function (isCoverage, isMapSdk, prefixDirectory) {
                 allowJs: true
             },
             coverageOptions: {
+                instrumentation: isCoverage ? true : false,
```

Comment:
This is required to debug locally when running `yarn test-browser`

Reply:
```suggestion
                instrumentation: isCoverage,
```